app/views/events/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container">
+<h2>Past Events</h2>
+<% @event.each do |event| %>
+  <% if event.date < Time.now %>
+      <h4><%= event.name %></h4>
+      <p><%= event.description %></p>
+      <span><%= event.creator.name %></span>
+      <h5><%= event.date%></h5>
+  <% end %> 
+<% end %>
+</div>
+
+<div class="container">
+<h2>Upcoming Events</h2>
+<% @event.each do |event| %>
+  <% if event.date > Time.now %>
+      <h4><%= event.name %></h4>
+      <p><%= event.description %></p>
+      <span><%= event.creator.name %></span>
+      <h5><%= event.date%></h5>
+  <% end %> 
+<% end %>
```

Comment:
In your events controller `#index` method you already have `@previous_events = Event.previous_events` defined, just use `@upcoming_events = Event.upcoming_events ....` and loop through all upcoming events